### PR TITLE
ML extension - View models styles / layout updates

### PR DESCRIPTION
**Pull request by halerankin**
Revised CSS styles and added modelBuilder components to the view models stage of the **View and import models**. Includes an SVG icon for the empty state -- when nothing is selected.

_screenshot_
image

_Figma comp_
image

This PR revises styles per [3914](https://mssqltools.visualstudio.com/SQL%20Tools%20Semester%20Work%20Tracking/_workitems/edit/3914/)

### CODE REVIEW

File: extensions/machine-learning/src/views/models/tableSelectionComponent.ts [hunk 1]
```diff
@@ -53,10 +54,10 @@ export class TableSelectionComponent extends ModelViewBase implements IDataCompo
 	 */
 	public registerComponent(modelBuilder: azdata.ModelBuilder): azdata.Component {
 		this._databases = modelBuilder.dropDown().withProperties({
-			width: this.componentMaxLength,
+			width: '221px'//this.componentMaxLength,
```
Comment (llali) [inline extensions/machine-learning/src/views/models/tableSelectionComponent.ts:57]: please remove the commented line

File: extensions/machine-learning/src/views/models/manageModels/currentModelsComponent.ts [hunk 2]
```diff
@@ -58,53 +63,84 @@ export class CurrentModelsComponent extends ModelViewBase implements IPageView {
...
+		}).component();
+		this._subheadingTextComponent = modelBuilder.text().withProperties(<azdata.CheckBoxProperties>{
+			value: constants.viewImportModelsDesc,
+			CSSStyles: {
+				'font-size': '13px'
+			}
+		}).component();
+		this._subheadingLinkComponent = modelBuilder.hyperlink().withProperties({
+			label: constants.learnMoreLink,
+			url: 'https://www.microsoft.com/',
```
Comment (llali) [inline extensions/machine-learning/src/views/models/manageModels/currentModelsComponent.ts:99]: the link is not correct. should we atleast link to main doc of ml until the doc is ready?

File: extensions/machine-learning/src/views/models/tableSelectionComponent.ts [hunk 3]
```diff
@@ -17,7 +17,8 @@ export interface ITableSelectionSettings {
 	databaseTitle: string,
 	tableTitle: string,
 	databaseInfo: string,
-	tableInfo: string
+	tableInfo: string,
+	layout?: string,
```
Comment (llali) [inline extensions/machine-learning/src/views/models/tableSelectionComponent.ts:21]: can you please make sure the page in predict wizard which using this component doesn't break?

**Review by llali — APPROVED** [review]
:shipit:

File: extensions/machine-learning/src/views/models/manageModels/currentModelsComponent.ts (hunk 2 above)
Comment (halerankin) [reply extensions/machine-learning/src/views/models/manageModels/currentModelsComponent.ts:99]: Placeholder until I know differently.

File: extensions/machine-learning/src/views/models/tableSelectionComponent.ts (hunk 3 above)
Comment (halerankin) [reply extensions/machine-learning/src/views/models/tableSelectionComponent.ts:21]: @llali I checked and the layout behaves the same in predict wizard.

**Pull request merged**